src/app/hobby-form/hobby-form.component.css

```diff
@@ -0,0 +1,42 @@
+.hobbies__container {

```

Comment:
as angular styles are incapsulated BEM is not needed, can use simpler class names (still up to you)